Project/Project/Project-Info.plist

```diff
@@ -36,5 +40,18 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSAppTransportSecurity</key>
```

Comment:
Good spot. This must have been automatically added at some point. Fixed in 4c4bb67.